Lib/dis.py

```diff
@@ -1 +1,19 @@
-from _dis import *
+import argparse
```

Comment:
Since there's also an `import argparse` inside the function, this can be removed

Reply:
Oooops, right. Thanks for the good catch!